addons/website/models/website_snippet_filter.py

```diff
@@ -141,7 +149,7 @@ def _filter_records_to_dict_values(self, records):
                 elif ('ir.qweb.field.%s' % field_widget) in self.env:
                     data['fields'][field_name] = self.env[('ir.qweb.field.%s' % field_widget)].record_to_html(record, field_name, {})
                 else:
-                    data['fields'][field_name] = record[field_name]
+                    data['fields'][field_name] = escape(record[field_name])
```

Comment:
why not, but what is the usecase ? Why do you need to add this ? and why in 14 and not in 13 if a security issue ?

Reply:
Indeed, I now see that the `else:` should never be reached because `field_widget` defaults to the field's `type`.
So, it can only happen if the template for that `type` is not available in the database - of if there is a typo in the `type` override.
In that case maybe an error should be raised to guide the user into fixing it rather than defaulting to some arbitrary formatting.